app/brave_generated_resources.grd

```diff
@@ -6,6 +6,62 @@
       <emit emit_type='prepend'></emit>
     </output>
     <output filename="brave_generated_resources.pak" type="data_package" />
```

Comment:
Should we generate `brave_generated_resources.pak`?

Reply:
yes, it's needed for the `includes`, otherwise, the `includes` for google tag manager polyfill and google services fails to load.